doc/source/dev/contributor/quickstart_pip.rst

```diff
@@ -0,0 +1,71 @@
+:orphan:
+
+.. _quickstart-pip:
+
+=======================================================================
+Development environment quickstart guide using ``pip`` on Ubuntu Linux)
+=======================================================================
+
+This is a high-level overview of what is needed to set up a development
+environment. This is only one possible way out of many. This guide assumes
+you have a fresh install of Ubuntu Linux 20.04, which only has a ``python3``
+executable. We also assume you have already installed ``git`` and cloned 
+the SciPy repository.
+
+Start with installing ``pip``::
+
+    sudo apt install -y python3-pip
+
+All further work should proceed in a virtual environment. Popular options include
+the standard library ``venv`` module or a separate 
+``virtualenv`` package. There are muliple third-party tutorials on how to
+set up a virtual environment, so we do not cover it here. 
+The rest of this guide assumes you use the ``virtualenv`` package and its
+companion ``virtualenvwrapper``. (You will also need to add several
+incantations to your ``.bashrc`` file.)::
```

Comment:
What do you mean by "incantations" in the `.bashrc`? I assume you do not mean the `pip3` command that follows?

Reply:
OK, fair enough, the phrasing is confusing. Is the updated version clearer?